packaging/linux/run_keybase

```diff
@@ -47,7 +47,7 @@ echo Mounting /keybase...
 kbfsfuse -debug -mdserver mdserver.kbfs.keybase.io:443 \
   -bserver bserver.kbfs.keybase.io:443 /keybase &>> "$logdir/keybase.kbfs.log" &
 echo Launching Keybase GUI...
-/opt/keybase/Keybase &>> "$logdir/keybase.gui.log" &
```

Comment:
It would be nice if we could have bash etc. get this value from the centralized constant too. But note that the value of `$logdir` above involves some logic that needs to be duplicated anyway. So it would only be a partial sharing at best, and parsing apart Go files is annoying.